src/surface/rov_gazebo/launch/sim_launch.py

```diff
@@ -20,9 +20,7 @@ def generate_launch_description() -> LaunchDescription:
     params_path = str(Path(rov_gazebo_path) / 'config' / params_file)
 
     # TODO: gz_sim launch might be nice
-    start_gazebo = ExecuteProcess(
-        cmd=['ign', 'gazebo', '-v', '3', '-r', world_path], output='screen'
-    )
+    start_gazebo = ExecuteProcess(cmd=['gz', 'sim', '-v', '3', '-r', world_path], output='screen')
```

Comment:
Did you check out gz_sim launch?

Reply:
Hmm I didn't know this existed but it looks pretty nice
https://index.ros.org/p/ros_gz_sim/